reframe/REFRAME_KNOWN_METHODS.md

```diff
@@ -203,6 +208,39 @@ Response:
 }}
 ```
 
+### FindProviders Blinded
+
+A message for finding nodes with interest in a given key using double hashing to blind the key being requested.
+
+```ipldsch
+    type FindHashedSHA256Request struct {
+        Query Bytes
+    }
+```
+
+The query is a derived hash of the multihash being requested.
+It is constructed by taking the raw bytes of the multihash, prepending the ascii bytes "CR_DOUBLEHASH", and taking the SHA256 hash of that data.
+The resulting digest is then packed itself into a multihash, using the multihash code identifier multihash.DBL_SHA2_256.
+
+The full semantics of double hashing in the context of content routing are described at https://www.notion.so/protocollabs/IPFS-Double-Hashing-Repurpose-8fdaae8748414ae592a5d24d59c0d8ed

```

Comment:
The notion page was migrated [here](https://www.notion.so/pl-strflt/Double-Hashing-for-Privacy-ff44e3156ce040579289996fec9af609)

Reply:
The page is not publicly available: 

> ![2022-10-17_23-04](https://user-images.githubusercontent.com/157609/196283125-d8e598c1-0930-4eec-a509-02cb3a58cf5c.png)

Reply:
Sorry, this link should work https://pl-strflt.notion.site/Double-Hashing-for-Privacy-ff44e3156ce040579289996fec9af609